extensions/git/package.json

```diff
@@ -391,6 +391,11 @@
         "command": "git.stashApplyLatest",
         "title": "%command.stashApplyLatest%",
         "category": "Git"
+      },
+      {
+        "command": "git.dropStash",
```

Comment:
shouldn't this be `stashDrop` to be consistent with the other commands? (eg. `git.stashPop`, `git.stashApply`, ...)

Reply:
Good point, I've updated the naming scheme 👍 .

Reply:
Did you forget to push this?